pkg/ruler/rules/local/local_test.go

```diff
@@ -0,0 +1,69 @@
+package local
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+	"time"
+
+	"github.com/prometheus/common/model"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v2"
+
+	rulefmt "github.com/cortexproject/cortex/pkg/ruler/legacy_rulefmt"
+	"github.com/cortexproject/cortex/pkg/ruler/rules"
+)
+
+func TestLoadingRules(t *testing.T) {
```

Comment:
I would justbe more clear that it's testing only `ListAllRuleGroups()`.
```suggestion
func TestClient_ListAllRuleGroups(t *testing.T) {
```